### PR TITLE
Update navigation bar to support dark mode

### PR DESCRIPTION
**Pull request by alwx**
Makes nav bar look properly with dark mode.

Screenshot 2023-07-24 at 12 10 49

#### Platforms

- Android
- iOS

##### Functional

- 1-1 chats
- public chats
- group chats

status: ready

### CODE REVIEW

**Review by flexsurfer — APPROVED** [review]

File: src/status_im2/contexts/chat/messages/navigation/style.cljs [hunk 1]
```diff
@@ -16,13 +16,13 @@
...
 
-(def background-view
+(defn background-view []
   {:position         :absolute
    :top              0
    :left             0
    :right            0
    :height           navigation-bar-height
-   :background-color (colors/theme-colors colors/white-opa-70 :transparent)
+   :background-color (colors/theme-colors colors/white-opa-70 colors/neutral-100-opa-70)
```
Comment (J-Son89) [inline src/status_im2/contexts/chat/messages/navigation/style.cljs]: while you're making changes here can you update this component to use `quo2.theme` with-theme HOC and then pass in `theme` as a 3rd param. e,g

```
(defn background-view [theme]
  {:position         :absolute
   :top              0
   :left             0
   :right            0
   :height           navigation-bar-height
   :background-color (colors/theme-colors colors/white-opa-70 colors/neutral-100-opa-70 theme)
   ```

**Review by rasom — APPROVED** [review]

**Review by vkjr — APPROVED** [review]

**Comment by alwx** [comment]
@J-Son89 done!

**Review by J-Son89 — APPROVED** [review]
Nice @alwx! 😀

**Comment by VolodLytvynenko** [comment]
hi, @Francesca-G could you check if the top bar is suitable on the dark mode? No issues from my side
design is here: https://www.figma.com/file/7KIYbhoqNGAIFonE0w9TDz/Messages-for-Mobile?type=design&node-id=3752-693872&mode=design&t=e58PXRjof6cefuXY-0

**Review by Francesca-G — COMMENTED** [review]
@VolodLytvynenko the blur doesn't look right (see screenshots). 
I'm aware that the blur on Android is a known issue but I tested it too.

iOS
2F2FFA01-5AFD-41DF-AEC2-C5D1BAF1653A_1_102_o

Android
C7079340-3E00-413A-BD5C-6EE0CCB8720A

Hope this helps!

**Comment by VolodLytvynenko** [comment]
> @VolodLytvynenko the blur doesn't look right (see screenshots). I'm aware that the blur on Android is a known issue but I tested it too.
> 
> iOS 2F2FFA01-5AFD-41DF-AEC2-C5D1BAF1653A_1_102_o
> 
> Android C7079340-3E00-413A-BD5C-6EE0CCB8720A
> 
> Hope this helps!

@Francesca-G. That's interesting. The bar looks different for you. 

This is how it looks for me:
Android:
image

IOS:
image


It seems that our builds are different. Could you please check if the previous PR build was removed to avoid the current last PR build installation over the previous one?

If it does not help, could you provide the model of devices you use?

**Review by Francesca-G — COMMENTED** [review]
@VolodLytvynenko thanks for helping me fix the builds, it was a problem on my side!

Here's the [Figma frame](https://www.figma.com/file/Tf5nfkYvpbnNCo4rKLK7lS/Feedback-for-Mobile?type=design&node-id=3614%3A306079&mode=design&t=ASz6Y0P5P9QgsQmS-1) with the design review

**Comment by VolodLytvynenko** [comment]
> @VolodLytvynenko thanks for helping me fix the builds, it was a problem on my side!
> 
> Here's the [Figma frame](https://www.figma.com/file/Tf5nfkYvpbnNCo4rKLK7lS/Feedback-for-Mobile?type=design&node-id=3614%3A306079&mode=design&t=ASz6Y0P5P9QgsQmS-1) with the design review

@Francesca-G thank you for the feedback!

Hi @alwx 
The top bar looks much better. The comments that Francesca described appear to be outside the scope of this PR. Let me know if you can fix them within the scope of this PR; otherwise, the PR can be merged and follow-ups for Francesca's comments can be created separately

**Pull request merged**